packages/commercetools/api-client/__tests__/setup.ts

```diff
@@ -7,7 +7,7 @@ jest.mock('apollo-client');
 jest.mock('@commercetools/sdk-auth', () => jest.fn(() => ({
   anonymousFlow: () => Promise.resolve('anonymous flow'),
   customerPasswordFlow: () => Promise.resolve('customer password flow'),
-  refreshTokenFlow: () => Promise.resolve('refresh token flow')
+  refreshTokenFlow: (refreshToken) => Promise.resolve(`refresh token flow ${refreshToken}`)
```

Comment:
Wouldn't it be enough just to resolve to `refreshToken`?

```suggestion
  refreshTokenFlow: (refreshToken) => Promise.resolve(refreshToken)
```